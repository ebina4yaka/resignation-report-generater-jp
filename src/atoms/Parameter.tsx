import React from 'react'
import Typography from '@material-ui/core/Typography'

type Props = {
  name: string
  value: string
}

export default function Parameters(props: Props): React.ReactElement {
  const { name, value } = props
  return (
    <div>
      <Typography variant="h6">{name}</Typography>
      <Typography variant="body1" gutterBottom>
        {value}
      </Typography>
    </div>
  )
}
