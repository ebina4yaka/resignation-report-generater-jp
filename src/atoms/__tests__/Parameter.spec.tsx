import React from 'react'
import renderer from 'react-test-renderer'
import Parameter from '../Parameter'

test('Snapshot test for Parameter', () => {
  const tree = renderer
    .create(<Parameter name="test name" value="test value" />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
