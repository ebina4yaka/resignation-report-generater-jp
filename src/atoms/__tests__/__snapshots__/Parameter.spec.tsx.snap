// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for Parameter 1`] = `
<div>
  <h6
    className="MuiTypography-root MuiTypography-h6"
  >
    test name
  </h6>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
  >
    test value
  </p>
</div>
`;
